config/materials/iso3506_a2-70.mtl

```diff
@@ -0,0 +1,140 @@
+﻿ND_RelParSet_K01 = {
+
+Name = ISO3506_A2-70
+
+PARAMETERS =
+{
+  Name = PTC_MATERIAL_TYPE
+  Type = Integer
+  Default = 9
+  Access = Locked
+},
+{
+  Name = PTC_FAILURE_CRITERION_TYPE
+  Type = String
+  Default = 'NONE'
+  Access = Locked
+},
+{
+  Name = PTC_FATIGUE_TYPE
+  Type = String
+  Default = 'NONE'
+  Access = Locked
+},
+{
+  Name = PTC_MATERIAL_SUB_TYPE
+  Type = String
+  Default = 'LINEAR'
+  Access = Locked
+},
+{
+  Name = PTC_MATERIAL_DESCRIPTION
+  Type = String
+  Default = 'Sample materials data from ANSYS Granta
+See grantadesign.com/PTC for more'
+  Access = Full
+},
+{
+  Name = GRANTA_DATALINK_VERSION
+  Type = String
+  Default = '5'
+  Access = Full
+},
+{
+  Name = GRANTA_DATABASE_KEY
+  Type = String
+  Default = 'Granta_Typical_Materials'
+  Access = Full
+},
+{
+  Name = GRANTA_IDENTITY_GUID
+  Type = String
+  Default = '46ef4dd7-2d17-4d2c-b5a0-bd890ebb0f27'
+  Access = Full
+},
+{
+  Name = GRANTA_VERSION_GUID
+  Type = String
+  Default = '43ddc23e-6caf-4833-9c7b-b0a33ab3fc1a'
+  Access = Full
+},
+{
+  Name = GRANTA_MODEL_REFERENCE
+  Type = String
+  Default = '00000000-0000-0000-0000-000000000000'
+  Access = Full
+},
+{
+  Name = PTC_MASS_DENSITY
+  Type = Real
+  Default = 7.900000e-06 kg/mm^3
+  Access = Full
+},
+{
+  Name = PTC_YOUNG_MODULUS
+  Type = Real
+  Default = 1.980000e+08 kPa
+  Access = Full
+},
+{
+  Name = PTC_POISSON_RATIO
+  Type = Real
+  Default = 2.700000e-01
+  Access = Full
+},
+{
+  Name = PTC_THERMAL_CONDUCTIVITY
+  Type = Real
+  Default = 1.500000e+04 mm kg /(sec^3 C)
+  Access = Full
+},
+{
+  Name = PTC_THERMAL_EXPANSION_COEF
+  Type = Real
+  Default = 1.700000e-05 /C
+  Access = Full
+},
+{
+  Name = PTC_SPECIFIC_HEAT
+  Type = Real
+  Default = 5.100000e+08 mm^2/(sec^2 C)
+  Access = Full
+},
+{
+  Name = PTC_TENSILE_YIELD_STRESS
+  Type = Real
+  Default = 7.000000e+02 MPa
+  Access = Full
+},
+{
+  Name = TEMPERATURE
+  Type = Real
+  Default = 0.000000e+00 C
+  Access = Full
+},
+{
+  Name = PTC_INITIAL_BEND_Y_FACTOR
+  Type = Real
+  Default = 5.000000e-01
+  Access = Full
+},
+{
+  Name = PTC_BEND_TABLE
+  Type = String
+  Default = ''
+  Access = Full
+},
+{
+  Name = PTC_DEFAULT_APPEARANCE
+  Type = String
+  Default = 'steel-stainless'

```

Comment:
Can you check if we have this appearance and how it looks like?

Reply:
I set this appearance by myself !
is the creo default for stainless steel
![image](https://user-images.githubusercontent.com/53296021/147548680-ee67613e-de45-4507-838c-4a406756bd1a.png)

@fiorisi